sdk/core/azure-core/inc/azure/core/http/http.hpp

```diff
@@ -160,7 +216,7 @@ namespace Azure { namespace Core { namespace Http {
     /**
      * @brief Construct an #Azure::Core::Http::Request.
      *
-     * @param httpMethod HTTP method.
+     * @param httpMethod HttpMethod.
```

Comment:
Since you are fixing this already, let's do something a bit more meaningful, rather than being redundant.
We can copy/paste it across all such param docs.

How about this:

```suggestion
     * @param httpMethod The HTTP verb indicating the action this request is intended to take..
```

Reply:
I am just creating the link to the HttpMethod class:

![image](https://user-images.githubusercontent.com/24213737/119566698-4827a580-bd60-11eb-9e42-7fe2dab32ce7.png)

Reply:
That's fine, we can do a separate PR/fix for  the wording :)